test/vanilla/Expected/AcceptanceTests/ModelFlattening/modelflattening/models/_models.py

```diff
@@ -175,8 +173,8 @@ class FlattenParameterGroup(msrest.serialization.Model):
     :type description: str
     :param max_product_display_name: Display name of product.
     :type max_product_display_name: str
-    :ivar capacity: Capacity of product. For example, 4 people. Default value: "Large".
-    :vartype capacity: str
+    :param capacity: Capacity of product. For example, 4 people.
```

Comment:
[swagger](https://github.com/Azure/autorest.testserver/blob/main/swagger/model-flattening.json#L519). Since current behavior of enum is `modelAsString` defaults to `False`, and `required` defaults to `False`, this param is an optional constant with no default value

Reply:
ask tim about whether `modelAsString` defaults to false or true